src/jit/morph.cpp

```diff
@@ -8205,7 +8207,12 @@ void Compiler::fgMorphTailCall(GenTreeCall* call, void* pfnCopyArgs)
     call->gtCallMoreFlags |= GTF_CALL_M_VARARGS | GTF_CALL_M_TAILCALL | GTF_CALL_M_TAILCALL_VIA_HELPER;
     call->gtFlags &= ~GTF_CALL_POP_ARGS;
 
-#endif // _TARGET_*
+#elif defined(_TARGET_ARM64_)
+    NYI("Tail calls via stub are unsupported on this platform.");
```

Comment:
These ARM64 specific NYIs do not hurt, but I would expect them to be required. Why are they not taken care of by `getTailCallCopyArgsThunk` returning null?

Reply:
There also is  NYI_ARM64()  which might be better to use here.